.changeset/smooth-vans-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-deployments': patch
```

Comment:
This'll be a `minor` bump since we're changing the API types, even though most users won't notice this.